File: servico-musica-java/java.soap/src/main/java/servico/java/soap/repository/PlaylistRepository.java
package servico.java.soap.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import servico.java.soap.entity.Playlist;

import java.util.List;

@Repository
public interface PlaylistRepository extends JpaRepository<Playlist,Long> {
    List<Playlist> findByMusicasId(Long idMusica);
}

File: servico-musica-kotlin/kotlin.soap/src/main/kotlin/servico/kotlin/soap/repository/PlaylistRepository.kt
package servico.kotlin.soap.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import servico.kotlin.soap.entity.Playlist

@Repository
interface PlaylistRepository : JpaRepository<Playlist, Long> {
    fun findByMusicasId(idMusica: Long?): MutableList<Playlist?>?
}
